StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebView.swift

```diff
@@ -25,6 +26,16 @@ class ConnectComponentWebView: ConnectWebView {
     /// The current notification center instance
     private let notificationCenter: NotificationCenter
     
+    private let setterMessageHandler: OnSetterFunctionCalledMessageHandler = .init()
+    
+    let activityIndicator: ActivityIndicator = {
+        let activityIndicator = ActivityIndicator()
```

Comment:
Non-blocker: Can be part of a followup PR

Should we add an accessibility label like "Loading"?

There is an already localized string in StripeIdentity that could be migrated to StripeUICore and reused (then we don't need to add localized strings to this module):

https://github.com/stripe/stripe-ios/blob/5541e9b3/StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift#L15-L17

Our Lokalise upload script should automatically handle migrating strings between modules when uploading to Lokalise (Runbook: go/mobile-localizations)

https://github.com/stripe/stripe-ios/blob/5541e9b3/ci_scripts/upload_localized_strings_to_lokalise.sh#L57-L59